### PR TITLE
Update L.label to Leaflet 1.0.0-rc.2 L.Tooltip

### PR DESCRIPTION
**Pull request by brandonxiang**
When I used L.storage, I found that the measure function is not available. Therefore, I checked out L.Measureable. 

I dont know its dependencies.  I tested it with leaflet 0.7.7, L.editable 0.6.2 and L.label 0.2.1, and it does not work. Then, I tested it with leaflet 1.0.0-beta2, L.editable 0.7.0-beta1 and L.label 0.2.1, and also it fails.

Finally, I try to use 1.0.0-rc2 with L.Tooltip to replace L.label, and add a `examples/index.html` for testing.

image

### CODE REVIEW

File: Leaflet.Measurable.js [hunk 1]
```diff
@@ -102,11 +102,11 @@ L.Measurable = L.Editable.extend({
             var latlng, latlngs = e.layer._defaultShape();
             for (var i = 0; i < latlngs.length; i++) {
                 latlng = latlngs[i];
-                latlng.__vertex.hideLabel();
+                latlng.__vertex.closeTooltip();
             }
             if (latlng && latlng.__vertex) {
                 var length = L.GeoUtil.lineLength(map, e.layer._defaultShape());
-                latlng.__vertex.bindLabel(L.GeoUtil.readableDistance(length, this.getMeasureUnit()), {noHide: true}).showLabel();
+                latlng.__vertex.bindTooltip(L.GeoUtil.readableDistance(length, this.getMeasureUnit()), {permanent: true}).openTooltip();
```
Comment (yohanboniface) [inline Leaflet.Measurable.js]: AFAIK `openTooltip` is not needed, when `permanent: true` is given.

File: examples/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <script src="../node_modules/leaflet-editable/src/Leaflet.Editable.js"></script>
+    <script src="../Leaflet.Measurable.js"></script>
+    <script>
+		var map = L.map('map', {editable: true}).setView([22.5365321328,113.9173539538], 13);
+        L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+		    attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+		}).addTo(map);
+
+     map.addControl(new L.MeasureControl());
+     document.getElementsByClassName('leaflet-measure-toggle')[0].innerHTML= "Measure";
```
Comment (yohanboniface) [inline examples/index.html]: Why not adding measurable css instead?

File: package.json [hunk 3]
```diff
@@ -20,5 +20,9 @@
   "bugs": {
     "url": "https://github.com/umap-project/Leaflet.Measurable/issues"
   },
-  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme"
+  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme",
+  "dependencies": {
```
Comment (yohanboniface) [inline package.json]: I think it's better to warn about dependencies in the README instead.
We had a bunch of troubles when setting non stable dependencies in plugins.

**Comment by yohanboniface** [comment]
Excellent!
Thank you for this :)
I've made some comments on details.

File: Leaflet.Measurable.js (hunk 1 above)
Comment (brandonxiang) [inline Leaflet.Measurable.js]: I remove `openTooltip` in the next pr.

File: package.json (hunk 3 above)
Comment (brandonxiang) [inline package.json]: I will declare the dependencies and usage in readme.md .

File: examples/index.html (hunk 2 above)
Comment (brandonxiang) [inline examples/index.html]: Do you mean that I should write a icon in Leaflet.Measurable.css or index.html?

Comment (yohanboniface) [inline examples/index.html]: No sorry, I've read too fast, that's OK.

File: package.json (hunk 3 above)
Comment (yohanboniface) [inline package.json]: Sorry, I meant to also remove them from package.json. npm is a pain, and we are talking of front libs here, not server side ones.

**Comment by yohanboniface** [comment]
Thanks!

**Pull request merged**